blocks/image-columns/image-columns.css

```diff
@@ -137,10 +164,130 @@
     padding: 36px 0;
     margin: 0 auto;
   }
+
+  .thank-you .image-columns-wrapper {
+    max-width: var(--section-desktop-max-width);
+    padding: 36px 0;
+    margin: 0 auto;
+  }
 }
 
 @media (min-width: 1600px) { /* large desktop */
   .image-columns-wrapper {
     max-width: var(--section-large-desktop-max-width);
   }
 }
+
+.thank-you .image-columns-wrapper .image-columns {
+  position: relative;
+  padding-top: 20px;
+}
+
+@media (min-width: 768px) {
+  .thank-you .image-columns-wrapper .image-columns {
+    margin-bottom: 48px;
+  }
+}
+  
+.thank-you .image-columns-wrapper .image-columns::after{
+  content: "";
+  position: absolute;
+  border-right: 2px dashed #dedede;
+  top: 0;
+  bottom: 0;
+  left: 50%;
+}
+
+.thank-you .image-columns > div > .image-columns-txt-col {
+  flex: 0 1 50%;
+  border-radius: 8px;
+  width: 90%;
+  box-shadow: 0 3px 8px 0 rgb(0 0 0 / 10%);
+  padding: 24px 24px 24px 16px;
+  position: relative;
+  display: flex;
+  align-items: center;
+  z-index: 1;
+  background-color: #fff;
+}
+
+@media (min-width: 768px) {
+  .thank-you .image-columns > div > .image-columns-txt-col {
+    order: 0;
+  }
+}
+
+.thank-you .image-columns > div > .image-columns-txt-col::after {
+  content: "";
+    position: absolute;
+    width: 21px;
+    height: 21px;
+    left: calc(50% - 9px);
+    top: -40px;
+    background-color: #eb0000;
+    border-radius: 50%;
+    z-index: 1;
+}
+
+.thank-you .image-columns > div:first-child > .image-columns-txt-col::after {
+  top: -80px;
+}
+
+.thank-you .image-columns > div > .image-columns-txt-col:nth-child(even)::after {
+  left: calc(50% - 9px);
+  top: -40px;
+}
+
+
+
+@media (min-width: 768px) {
+  .thank-you .image-columns > div > .image-columns-txt-col::after {
+    top: calc(50% - 9px);
+    right: -11px;
+    left: unset;
+  }
+  
+  .thank-you .image-columns > div:first-child > .image-columns-txt-col::after {
+    top: calc(50% - 9px);
+    left: unset;
+  }
+
+  .thank-you .image-columns > div > .image-columns-txt-col:nth-child(even)::after {
+    left: -11px;
+    top: calc(50% - 9px);
+  }
+}
+
+.thank-you .image-columns > div > .image-columns-img-col .img-container {
+  max-width: 431px;
+  z-index: 1;
+}
+
+.thank-you .image-columns > div > .image-columns-txt-col h3 {
+  font-size: 40px;
```

Comment:
Can we stick to our existing font-sizes and font-weights for consistency reasons?

Reply:
That code is referring to numbers in red.

<img width="127" alt="Screenshot 2023-10-02 at 17 30 50" src="https://github.com/hlxsites/bitdefender/assets/166901/2b5e8aed-e610-4e27-ba0d-a255c8c59164">